src/tutorials/annotation.md

```diff
@@ -1,17 +1,21 @@
 # Annotating a BIDS dataset
 
-## What is annotation?
+## What is an annotation?
 
-Annotation refers to metadata that is directly associated with  data.
+Annotation refers to metadata that is directly associated with data.
 Without adequate annotation, your valuable shared data may be of limited
 use to other researchers and to you in the future.
 
 While the BIDS requirements for annotation are limited,
 BIDS supports a framework for inserting comprehensive
 data annotation at several levels in the dataset.
+In addition, BIDS supports annotations of events and subject characteristics using
```

Comment:
I would say 'For example' rather than 'In addition'

Reply:
Changed [here](https://github.com/bids-standard/bids-starter-kit/pull/335/commits/fb1b7d87083388d5969afa7c10e59cca0b88214c)